src/polymer/docs.ts

```diff
@@ -350,6 +352,10 @@ export function featureElement(features: FeatureDescriptor[]):
         'The properties reflected here are the combined view of all ' +
         'features found in this library. There may be more properties ' +
         'added via other libraries, as well.',
+    observers: [],
```

Comment:
Looks like we both attacked the same problem from different directions. Removed, and did some more strictNullChecks-related cleanup.